spec/requests/posts_spec.rb

```diff
@@ -1,10 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
+  describe 'Get page post #index' do
```

Comment:
- [ ] You have three failing tests. Kindly make sure all tests are passing. 

<img width="602" alt="Screen Shot 2022-09-10 at 08 54 58" src="https://user-images.githubusercontent.com/69011963/189468557-4908003c-a1f8-4050-ac45-1a6164d51273.png">

<img width="852" alt="Screen Shot 2022-09-10 at 08 55 08" src="https://user-images.githubusercontent.com/69011963/189468573-325a5ad2-b138-4c75-9a37-71785c6d23ae.png">

Reply:
Hello @Rustamxon7,

Kindly please be sure to run the command `bundle install` in order to install all the gems that the project needs to run properly because as you can see in the image below the test are run properly on my machine.

![image](https://user-images.githubusercontent.com/99298594/189492301-e0ccac45-bccc-4f55-852b-689278b3ceef.png)

Hope to hear soon from you!

Best regards.